### PR TITLE
docs/enterprise: add info about license flags 

### PR DESCRIPTION
**Pull request by zekker6**
- added information about new `-license` flags
- added more details about how to run enterprise binaries

### CODE REVIEW

File: docs/enterprise.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ and [the license](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master
 
 The use of VictoriaMetrics enterprise components is permitted in the following cases:
 
-- Evaluation use in non-production setups. Just download and run enterprise binaries or packages of VictoriaMetrics
-  components from usual places - [releases page](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) and [docker hub](https://hub.docker.com/u/victoriametrics).
+- Evaluation use in non-production setups. Please, request trial license [here](https://victoriametrics.com/products/enterprise/).
```
Comment (tenmozes) [inline docs/enterprise.md:23]: we need to add something like:

if you want to try the enterprise features you can request free trial <link>, we email you license key after submitting the form

File: docs/enterprise.md [hunk 2]
```diff
@@ -20,16 +20,16 @@ and [the license](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master
...
+  Download components from usual places - [releases page](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) and [docker hub](https://hub.docker.com/u/victoriametrics).
   Enterprise binaries and packages have `enterprise` suffix in their names.
 
 - Production use if you have a valid enterprise contract or valid permit from VictoriaMetrics company.
   [Contact us](mailto:info@victoriametrics.com) if you need such contract.
```
Comment (tenmozes) [inline docs/enterprise.md:28]: let's change email to sales@...

File: docs/enterprise.md [hunk 3]
```diff
@@ -61,3 +61,192 @@ On top of this, enterprise package of VictoriaMetrics includes the following imp
...
+
+In order to run VictoriaMetrics enterprise helm chart it is required to provide the license key via `license` value in `values.yaml` file 
+and adjust the image tag to the enterprise one. 
+
+For example, the following values file for [VictoriaMetrics single-node chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-single)
+is used to provide key in plain-text:
+```yaml
+server:
+  image:
+    tag: vX.Y.Z-enterprise
```
Comment (tenmozes) [inline docs/enterprise.md:138]: let's provide the working example with real version

File: docs/enterprise.md [hunk 4]
```diff
@@ -61,3 +61,192 @@ On top of this, enterprise package of VictoriaMetrics includes the following imp
...
+In order to provide key via existing secret, the following custom resource is used:
+```yaml
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMSingle
+metadata:
+  name: example-vmsingle
+spec:
+  retentionPeriod: "1"
+  license:
+    keyRef:
```
Comment (tenmozes) [inline docs/enterprise.md:196]: should we provide example of config map with specified name and key?

File: docs/enterprise.md [hunk 5]
```diff
@@ -61,3 +61,192 @@ On top of this, enterprise package of VictoriaMetrics includes the following imp
...
+      name: vm-key
+      key: license
+  image:
+    tag: vX.Y.Z-enterprise 
+```
+
+Example secret with license key:
```
Comment (tenmozes) [inline docs/enterprise.md:203]: should we provide example how to create secret having license in file|console? as far we distribute it as just text

File: docs/enterprise.md (hunk 3 above)
Comment (zekker6) [reply docs/enterprise.md:138]: Yep, the idea is once we will have the first version which supports new license keys all entries of `X.Y.Z` on this page will be replaced by a real version

File: docs/enterprise.md (hunk 5 above)
Comment (zekker6) [reply docs/enterprise.md:209]: good point, added

File: docs/enterprise.md [hunk 6]
```diff
@@ -61,3 +62,202 @@ On top of this, enterprise package of VictoriaMetrics includes the following imp
...
+Alternatively, the license can be specified via `-license-file` command-line flag:
+```console
+./victoria-metrics-prod -license-file=/path/to/license/file
+```
+
+The license file must contain the license key.
+
+### Docker images
+
+Docker images for VictoriaMetrics enterprise are available [here](https://hub.docker.com/u/victoriametrics).
```
[the remark this answers is not in the document]
Comment (zekker6) [reply docs/enterprise.md:111]: added

**Pull request merged**